content/agaia.md

```diff
@@ -12,7 +12,7 @@ contributors: Matthew McCarthy
 
 Agaia is an organization that provides coliving retreats for impact-driven entrepreneurs, nomads, and communities in Italy.
 
-![[content/assets/agaia.jpg]]
```

Comment:
@matthewmccarthy11 have you got *relative* files paths set! How can you have content in the url if content is the root folder in your vault??

Reply:
Yeah its relative path to file, and the only reason I had /content/assets/name.jpg is because it didn't work when I tried assets/name.jpg and afterwards tried it with content in the front to see if it would work